### ISSUE TITLE
CentOS/RHEL7 doesn't build

### ISSUE THREAD

**Issue by Stig124** [jarun/nnn#1042]
#### Environment details (Put `x` in the checkbox along with the information)

- [x] Operating System: CentOS/RHEL7

#### Exact steps to reproduce the issue

OBS build logs (same for both CentOS and RHEL) :

```
[    1s] + exec rpmbuild -ba --define '_srcdefattr (-,root,root)' --nosignature /home/abuild/rpmbuild/SOURCES/nnn.spec
[    1s] Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.mh9OqX
[    1s] + umask 022
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + rm -rf nnn-4.1
[    1s] + /usr/bin/gzip -dc /home/abuild/rpmbuild/SOURCES/v4.1.tar.gz
[    1s] + /usr/bin/tar -xf -
[    1s] + STATUS=0
[    1s] + '[' 0 -ne 0 ']'
[    1s] + cd nnn-4.1
[    1s] + /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
[    1s] + exit 0
[    1s] Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.GKpzE8
[    1s] + umask 022
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + cd nnn-4.1
[    1s] + export 'CFLAGS=-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic'
[    1s] + CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic'
[    1s] + make -j16
[    1s] cc  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -std=c11 -Wall -Wextra -Wshadow -O3    -o nnn src/nnn.c -lreadline -lncursesw   -lpthread
[    1s] src/nnn.c:455:16: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'volatile'
[    1s]  static _Atomic volatile ulong_t num_files;
[    1s]                 ^
[    1s] src/nnn.c: In function 'dentfill':
[    1s] src/nnn.c:5101:3: error: 'num_files' undeclared (first use in this function)
[    1s]    num_files = 0;
[    1s]    ^
[    1s] src/nnn.c:5101:3: note: each undeclared identifier is reported only once for each function it appears in
[    1s] src/nnn.c: In function 'statusbar':
[    1s] src/nnn.c:5755:10: error: 'num_files' undeclared (first use in this function)
[    1s]           num_files, (ulong_t)pent->blocks << blk_shift, ptr);
[    1s]           ^
[    1s] make: *** [nnn] Error 1
[    1s] error: Bad exit status from /var/tmp/rpm-tmp.GKpzE8 (%build)
```
Command used ; `make -j16`

**Comment by jarun** [comment]
RHEL 7 problem is probably https://github.com/redis/redis/issues/6286.

Does CentOS 8 build?

**Comment by Stig124** [comment]
It does

**Comment by jarun** [comment]
Please drop RHEL 7 and whatever does not build. I am expecting those are dated.

**Comment by jarun** [comment]
Fixed at commit ccce80a1e91b99c58d1a4e1453b5238036e84f22.
